docs/usage.md

```diff
@@ -287,3 +288,104 @@ podman build . \
 ```
 
 Details: [write the Containerfile](#write-the-dockerfile-or-containerfile), [build the container](#build-the-container)
+
+
+## Example: cargo
+
+### Prefetch dependencies (cargo)
+
+```shell
+mkdir -p playground/pure-rust
+cd playground/pure-rust
+git clone git@github.com:sharkdp/bat.git --branch=v0.22.1
+cachi2 fetch-deps --source bat '{"type": "cargo"}'
```

Comment:
Calling this "pure-rust" because tomorrow I'm going to add a different use case: python with rust dependencies (👀 cryptography 👀 )